distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -391,6 +391,14 @@ grant codeBase "file:/security-sts-attributequeryclaimshandler/security-sts-ldap
     permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}attributeMap.properties", "read";
 }
 
+grant codeBase "file:/org.apache.karaf.jaas.command" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}users.properties", "read";
+}
+
+grant codeBase "file:/org.apache.karaf.jaas.modules" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}users.properties", "write";
+}
```

Comment:
❗️ There's already a grant block for this module that you should just add the write permission to. https://github.com/blen-desta/ddf/blob/5b384c1f7e9a39f00e4b38bb7c5b543f54585e46/distribution/ddf-common/src/main/resources/security/default.policy#L185